app/AppComponents.scala

```diff
@@ -62,14 +64,14 @@ class AppComponents(context: Context, identity: AppIdentity)
   /**
     * Set up matchers and add them to the matcher pool as the app starts.
     */
-  def initialiseMatchers = {
+  def initialiseMatchers: Future[Unit] = {
     for {
-      (rules, _) <- ruleResource.fetchRulesByCategory()
+      (rulesByCategory, _) <- ruleResource.fetchRulesByCategory()
     } yield {
-      rules.foreach { case (category, rules) => {
-        val (matcher, _) = languageToolFactory.createInstance(category.name, MatcherConfig(rules))
+      rulesByCategory.foreach { case (category, rules) => {
+        val matcher = new RegexMatcher(category.name, rules)
```

Comment:
Just checking my understanding here: of the box we get regex matchers for all the entries in the google sheet?

Reply:
For each category. So, not a matcher per rule.